src/styles/StyleUtils.ts

```diff
@@ -61,48 +94,61 @@ const avatarSizes = {
     [CONST.AVATAR_SIZE.SMALL_NORMAL]: variables.avatarSizeSmallNormal,
 };
 
-const emptyAvatarStyles = {
+const emptyAvatarStyles: Partial<Record<AvatarSizesValues, ViewStyle | CSSProperties>> = {
```

Comment:
Maybe we can also have a type for the keys of this record like EmptyAvatarSizeName?

Reply:
I didn't get very far in my review, I'll do more later.

Reply:
Done!